src/main/webapp/app/exercises/shared/feedback/group/programming-feedback-groups.ts

```diff
@@ -22,8 +22,8 @@ class ProgrammingFeedbackGroupWrong extends FeedbackGroup {
     }
 
     shouldContain(feedbackItem: FeedbackItem): boolean {
-        const isReviewerFeedback = feedbackItem.type === 'Reviewer' && feedbackItem.credits && feedbackItem.credits < 0;
-        const isTestFeedback = feedbackItem.type === 'Test' && feedbackItem.credits !== undefined && feedbackItem.credits <= 0;
+        const isReviewerFeedback = feedbackItem.type === 'Reviewer' && feedbackItem.positive !== undefined && !feedbackItem.positive;
+        const isTestFeedback = feedbackItem.type === 'Test' && feedbackItem.positive !== undefined && !feedbackItem.positive;
         return isReviewerFeedback || isTestFeedback;
```

Comment:
```suggestion
        const isWrong = feedbackItem.positive !== undefined && !feedbackItem.positive;
        const isReviewerFeedback = feedbackItem.type === 'Reviewer';
        const isTestFeedback = feedbackItem.type === 'Test';
        return isWrong && (isReviewerFeedback || isTestFeedback);
```

Reply:
doesn't work as reviewer feedback needs to rely on `credits`